### PR TITLE
hooks: create xdg-email as a copy of xdg-open

### PR DESCRIPTION
**Pull request by soumyaDghosh**
Many apps, specifically browsers like chromium, brave doesn't work with xdg-open only to open mailto links. This would help those apps to be able to open mailto links.
Also can revert this commit, https://git.launchpad.net/review-tools/commit/?id=31c894178a8fd110b9f65f0ee532b7e11384f8b8

### CODE REVIEW

File: hooks/500-create-xdg.chroot [hunk 1]
```diff
@@ -29,6 +29,8 @@ x-scheme-handler/mailto=xdg-open.desktop
 x-scheme-handler/help=xdg-open.desktop
 EOF
 
+cp /usr/bin/xdg-open /usr/bin/xdg-email
```
Comment (Meulengracht) [inline hooks/500-create-xdg.chroot:32]: could it have been a symlink?

Comment (alfonsosanchezbeato) [reply hooks/500-create-xdg.chroot:32]: Also, it does not look like they are the same binary in fact, not in noble.

Comment (alfonsosanchezbeato) [reply hooks/500-create-xdg.chroot:32]: Oh, but I see `xdg-open` is a script we create in this hook nevermind.

**Review by alfonsosanchezbeato — COMMENTED** [review]
Probably this is fine, but please create a symlink instead and do it nearer to where we write `/usr/bin/xdg-open`.

**Comment by alfonsosanchezbeato** [comment]
Also, please amend commit message so it is `hooks:...` instead of `feat:...`

**Comment by soumyaDghosh** [comment]
Sorry for wrong commit message, recreating this PR. Please forgive me for the inconveniences.

**Pull request closed without merging**